app/src/main/java/com/github/sdp/mediato/ui/viewmodel/MyFollowingViewModel.java

```diff
@@ -0,0 +1,36 @@
+package com.github.sdp.mediato.ui.viewmodel;
+
+import com.github.sdp.mediato.data.Database;
+import com.github.sdp.mediato.model.User;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class MyFollowingViewModel extends UserViewModel {
+    public void reloadUser() {
```

Comment:
Thanks for adding this method :) It's going to be pretty useful in the future.

Reply:
thanks :)